app/views/foods/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="container w-full p-4 b-2">
+    <h2 class="text mb-4">Add new food</h2>
+    <%= form_with model: @food, url: foods_path,  local: true, data: {turbo:false} do |f| %>
+        <div class="mb-3">
+            <%= f.label :name, class: "form-label" %>
+            <%= f.text_field :name, class: "form-control" %>
+        </div>
+        <div class="mb-3">
+            <%= f.label :measurement_unit, class: "form-label" %>
+            <%= f.text_field :measurement_unit, class: "form-control" %>
```

Comment:
- [ ] When I'm trying to create food, I'm getting this error. Please make sure to fix this issue.
![image](https://user-images.githubusercontent.com/83803414/195933414-cb77539d-36cf-404c-907f-a4d44c4dff18.png)

Reply:
Hello @thecodechaser! :wave: Thank you so much for your review.
The schema has a "user_id" foreign key in the foods table. I guess the migration file for it was deleted along the way. I recreated a migration to add user_id in the Foods table. Thanks for your review!:raised_hands:
![image](https://user-images.githubusercontent.com/88782702/195939955-99e7b0bc-f32d-4eb9-a452-60f47b164748.png)